src/shared/tool-groups.ts

```diff
@@ -25,7 +26,7 @@ export const TOOL_DISPLAY_NAMES = {
 // Define available tool groups
 export const TOOL_GROUPS: Record<string, ToolGroupConfig> = {
 	read: {
-		tools: ["read_file", "search_files", "list_files", "list_code_definition_names"],
+		tools: ["read_file", "fetch_instructions", "search_files", "list_files", "list_code_definition_names"],
```

Comment:
Hmm, I'm not sure this belongs in the `read` group, since I think this is about reading the user's files. Might want to create a new alwaysAvailable group? Don't feel strongly if it's complicated though.

Reply:
I replied to this elsewhere.  My thought was that it shared a characteristic with read_files of using up context window / token allowance.

But I can see there are privacy considerations for the read_file group that don't apply to fetch_instructions, so it's not a perfect alignment.  I will give it a bit more thought and suggest a way forward.